src/som/interpreter/actors/EventualMessage.java

```diff
@@ -112,7 +113,7 @@ protected AbstractDirectMessage(final Actor target, final SSymbol selector,
       this.sender = sender;
       this.target = target;
 
-      if (VmSettings.SNAPSHOTS_ENABLED) {
+      if (VmSettings.SNAPSHOTS_ENABLED && !VmSettings.REPLAY) {
```

Comment:
Seeing these and other flags, should we try to have two distinct: `SNAPSHOT_RECORDING` and `SNAPSHOT_REPLAY`? This could possibly avoid issues with wrongly combining/testing flags?

So, I mean, instead of having `SNAPSHOT_ENABLED`.

Reply:
Yes, refactoring the flags probably reduces ambiguity and improves readability of the code